core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -641,53 +645,115 @@ bool core_mmu_user_mapping_is_active(void)
 	return read_ttbr0() != read_ttbr1();
 }
 
-static paddr_t map_page_memarea(struct tee_mmap_region *mm)
+static void print_mmap_area(const struct tee_mmap_region *mm __maybe_unused,
+				const char *str __maybe_unused)
+{
+	if (!(mm->attr & TEE_MATTR_VALID_BLOCK))
+		debug_print("%s [%08" PRIxVA " %08" PRIxVA "] not-mapped",
+				str, mm->va, mm->va + mm->size);
+	else
+		debug_print("%s [%08" PRIxVA " %08" PRIxVA "] %s-%s-%s-%s",
+				str, mm->va, mm->va + mm->size,
+				mm->attr & (TEE_MATTR_CACHE_CACHED <<
+					TEE_MATTR_CACHE_SHIFT) ? "MEM" : "DEV",
+				mm->attr & TEE_MATTR_PW ? "RW" : "RO",
+				mm->attr & TEE_MATTR_PX ? "X" : "XN",
```

Comment:
"XN" is the ARM convention for eXecute-Never. I would prefer to use it instead of a "NX".
Maybe I should rename `CFG_CORE_RxDATA_NOEXEC` into `CFG_CORE_RxDATA_XN` ?